SKILL.tsv

```diff
@@ -5110,18 +5110,18 @@ SKILL_20150918_005109	$Duration: 30 seconds{nl}Zhendu Duration: #{CaptionRatio}#
 SKILL_20150918_005110	${#993399}{ol}[Magic] - [Holy]{/}{/}{nl}Creates a magic circle in front of you, healing allies or damaging enemies. Does not affect airborne enemies. Amount of HP recovered depends on target's maximum HP.
 SKILL_20150918_005111	$Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionTime}#%{nl}HP Recovery: 10% + #{CaptionRatio}# ~ #{CaptionRatio2}# HP{nl}Magic Circle Duration: 40 seconds
 SKILL_20150918_005112	$Summons a holy torch, improving the HP recovery of nearby allies. Decreases the accuracy of nearby enemies.
-SKILL_20150918_005113	{#993399}{ol}[Magic] - [Holy]{/}{/}{nl}Sprinkle holy water to increase the physical defense of you and your allies and to inflict damage on enemies.
+SKILL_20150918_005113	{#993399}{ol}[Magic] - [Holy]{/}{/}{nl}Sprinkle holy water to help increase the physical defense of you and your allies and to inflict damage to enemies.
 SKILL_20150918_005114	$Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio2}#%{nl}Physical Defense: +#{CaptionRatio}#%{nl}Targets: 10{nl}Duration: #{CaptionTime}# seconds{nl}Consumes 1 Holy Water
 SKILL_20150918_005115	$Maximum number of Magic Circles: #{SkillFactor}#{nl}Increases DEX for allies{nl}Enemy's Evasion & Defense: -#{CaptionRatio}#{nl} Duration: #{CaptionTime}# seconds
 SKILL_20150918_005116	$Mass Heal
-SKILL_20150918_005117	$Restores your HP and that of your allies in front of you. 
+SKILL_20150918_005117	$Restores the HP of you and that of your allies in front of you.
 SKILL_20150918_005118	$Restores #{CaptionRatio}#% of your maximum HP{nl}Additional HP Recovery: #{CaptionRatio2}#{nl}
 SKILL_20150918_005119	$Stone Skin
-SKILL_20150918_005120	$Temporarily increases the number of blocks of you and your party members.
+SKILL_20150918_005120	$Temporarily increase the number of blocks of you and your party members.
 SKILL_20150918_005121	$Block +#{CaptionRatio}#{nl}Duration: 35 seconds
-SKILL_20150918_005122	
-SKILL_20150918_005123	
-SKILL_20150918_005124	
+SKILL_20150918_005122	${#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Instruct your summoned zombies to circle dance. Zombies deal damage to anything they touch.
+SKILL_20150918_005123	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Maximum Duration: 10 seconds{nl}Requires Summoned Zombie
+SKILL_20150918_005124	Plant a Samedi glyph on the ground symbolizing a death noir. Movement speed and maximum HP are increased for you and your zombies inside the glyph's radius. The caster himself recieves only half the effect. 
```

Comment:
Change to: Plants a Samedi glyph on the ground symbolizing a death noir. Increases the maximum HP and movement speed of you and your zombies nearby the glyph. Only half of the effect applies to you. 

**Note** I don't know if it is Samedi or Samdi. This is from a Bokor skill named 'Samdiveve', so it might be Samdi